src/ape/api/transactions.py

```diff
@@ -259,3 +294,339 @@ def await_confirmations(self) -> "ReceiptAPI":
                 time.sleep(time_to_sleep)
 
         return self
+
+    def show_trace(self, verbose: bool = False, file: IO[str] = sys.stdout):
+        """
+        Display the complete sequence of contracts and methods called during
+        the transaction.
+
+        Args:
+            verbose (bool): Set to ``True`` to include more information.
+            file (IO[str]): The file to send output to. Defaults to stdout.
+        """
+        tree_factory = CallTraceParser(self, verbose=verbose)
+        call_tree = self.provider.get_call_tree(self.txn_hash)
+        root = tree_factory.parse_as_tree(call_tree)
+        console = RichConsole(file=file)
+        console.print(f"Call trace for [bold blue]'{self.txn_hash}'[/]")
+
+        if call_tree.failed:
+            default_message = "reverted without message"
+            if not call_tree.returndata.hex().startswith(
+                "0x08c379a00000000000000000000000000000000000000000000000000000000000000020"
+            ):
+                suffix = default_message
+            else:
+                decoded_result = decode_abi(("string",), call_tree.returndata[4:])
+                if len(decoded_result) == 1:
+                    suffix = f'reverted with message: "{decoded_result[0]}"'
+                else:
+                    suffix = default_message
+
+            console.print(f"🚫 [bold red]{suffix}[/]")
```

Comment:
Does this emoji render?

Reply:
I suppose it may depend on your Terminal, but it has for me everywhere.
Do we still want it?

Reply:
We don't really have emojis anywhere else, I think the color is fine